pull_alerts.py

```diff
@@ -65,12 +65,35 @@ def print_all_incidents(group_by_description=False):
         all_incidents.append(formatted_incident)
 
     all_incidents = sort_incidents(all_incidents, group_by_description)
+    print_stats(all_incidents, include_stats)
     prev_description = None
+    if include_incidents_as_blockquote:
+        print("""# Raw incident log
+```
+""")
     for incident in all_incidents:
         if group_by_description and incident.description != prev_description:
             prev_description = incident.description
             print("########### {} ##########\n".format(incident.description))
         print(incident.pretty_output())
+    if include_incidents_as_blockquote:
+        print("```")
+
+
+def print_stats(all_incidents, include_stats):
+    if not include_stats:
+        return
+    print("""# Statistics
+| Incidents            | Number |
+| -------------------- | ------ |
+| Total                | {} |
+| Actionable (#a)      | {} |
+| Non Actionable (#na) | {} |
+| Not Tagged           | {} |
+""".format(len(all_incidents),
+           len([i for i in all_incidents if actionable(i)]),
```

Comment:
Have you considered doing this in one loop through the incidents instead of a loop for each category?